tests/test_optimizers.py

```diff
@@ -215,3 +223,164 @@ class TestFtrlOptimizer(GPflowTestCase, OptimizerCase):
 # E               TypeError: Input 'lr' of 'ApplyProximalAdagrad' Op has type float32 that does not match type float64 of argument 'var'.
 #
 # ../../anaconda3/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py:546: TypeError
+
+def test_VGP_vs_GPR(session_tf):
```

Comment:
Just wanted to say that I absolutely love this test.

Reply:
It could do with a docstring though. Obvious to me, bizarre to the unitiated

Reply:
Agreed! @hughsalimbeni